data/language/nl-NL.yml

```diff
@@ -1968,22 +1968,22 @@ strings:
   1919: Use preferred owner name when starting a new game
   1920: "{SMALLFONT}{COLOUR BLACK}Select this option to use the same company owner name every time you start a new game"
   1921: "{COLOUR WINDOW_2}Preferred owner name: {COLOUR BLACK}{STRINGID}"
-  1922: Preferred Owner Name
-  1923: "Enter the preferred owner name:"
+  1922: Voorkeursnaam eigenaar
+  1923: "Voer de voorkeursnaam voor de eigenaar in:"
 
   1924: "{COLOUR BLACK}{TINYFONT}Power"
   1925: "{COLOUR BLACK}{TINYFONT}Brake"
   1926: "{SMALLFONT}{COLOUR BLACK}Move the selected order up in the list"
   1927: "{SMALLFONT}{COLOUR BLACK}Move the selected order down in the list"
   1928: Unable to run the game directly from read-only media
 
-  1929: Atari Inc. Credits...
+  1929: Credits voor Atari Inc....
```

Comment:
Strings 1929 and 1931 are no longer in use. This is indicated in `string_ids.h` using the `UNUSED_STR` macro. (We have not removed any strings yet, just in case.)

Reply:
I see. Should I remove those entries from nl-NL.yml entirely?

Reply:
Not yet; I'd prefer to do that for all languages at the same time when we get to it.